projects/canopy/src/lib/header/header.stories.ts

```diff
@@ -147,3 +148,46 @@ coBrandedHeader.parameters = {
     },
   },
 };
+
```

Comment:
Can you update the `argsTable` to show only the controls we want to view for this specific story please?